lib/workflow/stores/mongo.js

```diff
@@ -621,6 +622,7 @@ function mongoStoreFactory(waterline, Promise, Constants, Errors, assert, _, san
             state: {
                 $ne: Constants.Task.States.Succeeded
             },
+            ignoreFailure: false,
```

Comment:
Is it possible for this field to be `null` instead of `false` if not specified, preventing graphs without ignoreFailure tasks from succeeding?